templates/config/manifests/bases/clusterserviceversion.yaml.tpl

```diff
@@ -0,0 +1,73 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    category: "Cloud Provider"
+    alm-examples: '[]'
```

Comment:
this isn't a typo is it? "alm" not "olm"?

Reply:
Correct - this is not a typo: https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/building-your-csv.md#crd-templates